tests/test_aiokafka/test_aiokafka_consumer.py

```diff
@@ -143,7 +150,7 @@ async def test_getmany_with_limit_without_commit(self):
             topic=self.test_topic, partition=0, key="test2", value="test2"
         )
         self.consumer.subscribe(topics=[self.test_topic])
-
+        await self.consumer.start()
```

Comment:
I'd suggest preserving the vertical whitespace here -- it indicates a logical break between two separate chunks of code, each of which have different intent.

```suggestion
        await self.consumer.start()

```